11LessonEleven/in-class-project/css/style.css

```diff
@@ -0,0 +1,47 @@
+header{
+  background-color:
+#2BCDC0;
+text-align: center;
+height:400px;
+}
+section{
+  background-color:black;
+}
+h1{
+  padding-top:136px;
+}
+.box{
+  text-align: center;
+  color:white;
+  width:33%;
+  height:400px;
+  float:left;
+}
+  .clearfix:after {
+   content:"";
+   display:block;
+   clear:both;
+ }
+
+}
+@media max-width(768px){
```

Comment:
You are writing the `media query` incorrectly. That's why it's not working.
Review the lesson to get this fixed.